extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -176,6 +180,17 @@ class AmpDailymotion extends AMP.BaseElement {
     return this.loadPromise(this.iframe_);
   }
 
+  /** @override */
+  unlayoutCallback() {
```

Comment:
Agreed.

Reply:
Maybe. I'd be happy to move if that's where you are leaning. But to clarify: it's here because the `PauseHelper` should be released in the `unlayoutCallback` and then I noticed that here and in a couple of other places the unlayout callback was "forgotten".

Reply:
A set of separate pull requests have been filed just for the `unlayoutCallback`. They are related to this work, but it's a requirement that we've previously missed, so I agree it should go separately.